lib/mix/lib/mix/tasks/xref.ex

```diff
@@ -1003,10 +1003,12 @@ defmodule Mix.Tasks.Xref do
   end
 
   defp cycles(graph, opts) do
+    # Vertices order in cyclic_strong_components/1 return is arbitrary and changes between
+    # OTP versions, :desc sorting is necessary to make the output stable across versions.
     cycles =
       graph
       |> :digraph_utils.cyclic_strong_components()
-      |> Enum.reduce([], &inner_cycles(graph, &1, &2))
+      |> Enum.reduce([], &inner_cycles(graph, Enum.sort(&1, :desc), &2))
```

Comment:
Or perhaps I can use ascending order, and fix all tests?
Maybe I could sort the list of strong components as well, since its order is not guaranteed either.

Reply:
Any way is fine. Whatever you prefer.

Reply:
Went with ascending order after all, makes more sense I think [25098bc](https://github.com/elixir-lang/elixir/pull/13430/commits/25098bc5f13fa8ae4893aba9a9b26f1d4760772c)